ecosystem/sep-0006.md

```diff
@@ -80,7 +80,7 @@ SEP-6 lays out many options for how deposit and withdrawal can work. These are r
 
 ### Basic Wallet Implementation
 
-* Identify anchors you want to support manually, and test them with your wallet to be sure they work before whitelisting them. We encourage you to support as many anchors as possible.
+* Identify anchors you want to support manually, and test them with your wallet to be sure they work before allowing them to be used with your wallet. We encourage you to support as many anchors as possible.
```

Comment:
In several places: "Authorising" rather than "allowing" mores exactly replicates the original intent (it's less woolly). I don't feel strongly about this though.

Reply:
I avoided the word authorizing here because the term is used in relation to trustlines, where-as here we're talking about an application choosing to support an anchor in its product.